reader.py

```
@@ -15,9 +14,9 @@
 from metadata_gui_panel import MetadataDialog
 
 
-def sort_imgs(filename: str) -> int | None:
-    numbers = re.findall(r"\d+", filename)
-    return int(numbers[-1]) if numbers else -1
+def sort_imgs(filename: str) -> int:
```

Ruff (0.11.9)

Unused function argument: `filename`

(ARG001)